open_api_framework/conf/base.py

```diff
@@ -998,12 +997,16 @@ def init_sentry(before_send: Callable | None = None):
     + CORS_ALLOWED_ORIGINS
 )
 
-CSP_IMG_SRC = CSP_DEFAULT_SRC + config(
-    "CSP_EXTRA_IMG_SRC",
-    default=[],
-    split=True,
-    group="Content Security Policy",
-    help_text="Extra ``img-src`` sources for CSP other than ``CSP_DEFAULT_SRC``.",
+CSP_IMG_SRC = (
+    CSP_DEFAULT_SRC
+    + ["data:", "cdn.redoc.ly", "cdn.jsdelivr.net"]  # used by DRF spectacular
```

Comment:
Hmmm It looks like you use two solutions in the same time:
* `sidecar` to load assets locally
* whitelist redoc CDN for CSP

I think sidecar should be enough.

Reply:
Removed CDN links, does cause issues for the redoc logo which they do not include for "quick rebranding" 

OZ does use `raw.githubusercontent.com`

Reply:
I've read their solution for REDOC specifically and yeah, you were right, you need this CDN for the images
Sorry for my first comment